lambda/mixpanel.js

```diff
@@ -0,0 +1,15 @@
+const mixpanel = require("mixpanel-browser");
+
+// var mixpanel = require('mixpanel-browser');
+ 
+//mixpanel.init("YOUR_TOKEN");
+//mixpanel.track("An event");
+
+exports.mixpanelSend = async () => {
+
+    //initialize mixpanel with the given token (place into a secrets file)
+    mixpanel.init("token here");
```

Comment:
The mixpanel key will need to be set per-environment. You can use the build system to define per-environment variables that are included at build time. 

https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5

Reply:
doh, just realized this is the backend... so ignore my react link above. You can use the serverless.yml file to store environment variables that are set at deployment time (see people search code for similar examples)